packages/viz/src/components/common/stat/Stat.js

```diff
@@ -217,14 +243,9 @@ class Stat extends React.Component {
     const { renderer: Renderer, ...chartProps } = this.chartProps
 
     return (
-      <Collapse
```

Comment:
Why removing Collapse element ?

Reply:
Was not supported in react 17. I could not find what was broken without it, so I figured that everything was still working...

Reply:
The show must go on !!